cmd/containerd-shim-runhcs-v1/exec_hcs.go

```diff
@@ -467,6 +483,13 @@ func (he *hcsExec) waitForExit() {
 	he.exitedOnce.Do(func() {
 		close(he.exited)
 	})
+
+	if he.saveAsTemplate {
+		// Now save this host as at template
```

Comment:
can you update the comment for the function to include information about saving as a template?